### PR TITLE
Lschauer

### PR DESCRIPTION
**Pull request by LoganSchauer**

### CODE REVIEW

File: headers/lschauer.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Logan Schauer
+// Print a random color
+
+#include <iostream>
+
+void printRandColor() {
+	std::cout << "RED\m";
```
Comment (KrumphauChicken) [inline headers/lschauer.h]: That's not random. That's red.

Comment (LoganSchauer) [inline headers/lschauer.h]: My bad, I thought you meant print a random color of choice. Everything else is alright?

Comment (KrumphauChicken) [inline headers/lschauer.h]: Yes, but also label it in the menu in main.cpp.

Comment (LoganSchauer) [inline headers/lschauer.h]: Alright, I added it in the menu and hopefully the new header function is correct.

**Pull request merged**